### ISSUE TITLE
have any plans to make it multiplayer via webrtc?

### ISSUE THREAD

**Issue by jbenet** [bglusman/bughouse#1]

**Comment by bglusman** [comment]
Yes and no... I did have that plan when I started it at Recurse, and would still like to, but not sure if I have time to tackle now.  Was planning to move and/or duplicate the logic into node and maybe start with a ReactNative client... I meant the game itself to be react so I could test porting from one react framework to another, and there is a different chess game that's already done in react that might simplify this, but I just didn't have enough time/skill... there are some others at Recurse that were interested, and one was considering trying to add an AI able to play bughouse, which would be... interesting....  @jamak, @noahennis maybe others :-)  This code is pretty terrible though, it might be easier to start clean, or just take a few of the compatibility things I figured out, like my fork of the chess.js library to allow king capture.

**Comment by jbenet** [comment]
@bglusman thanks. i also found https://github.com/RogersN/bughouseChess which looks pretty good, too (works via websockets). i'll improve one soon
